pkg/api/norman/customization/project/project_actions.go

```diff
@@ -269,8 +269,21 @@ func (h *Handler) setPodSecurityPolicyTemplate(actionName string, action *types.
 	request *types.APIContext) error {
 	input, err := handler.ParseAndValidateActionBody(request, request.Schemas.Schema(&managementschema.Version,
 		client.SetPodSecurityPolicyTemplateInputType))
+	// In the empty body scenario, the content length is reported as 2.  We are using the content length
+	// to try and differentiate the ParseAndValidateActionBody failure scenarios.  If the body is just empty,
+	// we will treat it as an attempt to set it to nil, any other failure in parsing/validating will still
+	// result in an error.
+	const emptyBodyContentLength = 2
```

Comment:
Is this value set to 2 because we expect that the empty body is `"{}"`? If not, would you mind explaining why this is 2?

Reply:
Yeah.  I hate magic numbers like this, but I wasn't sure of another way to distinguish between parsing failing due to empty body vs parsing failure due to something else.